RadarSDK/RadarTimeZone.m

```diff
@@ -8,6 +8,16 @@
 #import "RadarTimeZone.h"
 #import "RadarUtils.h"
 
+NSDateFormatter *_timezoneDateFormatter = nil;
```

Comment:
Probably there no action items for with respect to this PR, but we really should consider having a more "central" place where we define our time parsers. There are quite a few definitions littered around the codebase. Just a call out to keep in mind for the future.